src/Fetchers/TMSFetcher/TMSFetcher.ts

```diff
@@ -75,41 +75,45 @@ export class TMSFetcher extends Fetcher {
      * Fetch the matches from TMS.
      */
     protected async fetch() {
```

Comment:
It doesn't seem like there are any changes to this function, except for a try-catch solution instead of promise-based. Please revert this to the original implementation, as it's more readable and does functionally the same.